pkg/grouper/grouper_models.go

```diff
@@ -14,6 +18,13 @@ func ConvertVulnerabilityToIDAliases(c []models.Vulnerability) []IDAliases {
 			ID:      v.ID,
 			Aliases: v.Aliases,
 		}
+
+		// For Debian Security Advisory data,
+		// all related CVEs should be bundled together, as they are part of this DSA.
+		if strings.Split(v.ID, "-")[0] == "DSA" {
```

Comment:
This feels like it could be inaccurate as it makes the DSA related transitive as well. I think it's fine for now but can you add a TODO here to revisit this implementation later.

Reply:
I.e. we probably want to add Related field to IDAliases, and move this logic into Grouper.

Reply:
Yeah, added a todo to provide a general way to handle all linux security advisories,